src/Collection/Collection.router.spec.ts

```diff
@@ -63,7 +58,30 @@ describe('Collection router', () => {
             { id: collectionAttributes.id, eth_address: wallet.address }
           )
         })
-        .finally(() => jest.useRealTimers())
+    })
+
+    it('it should fail with an error if the update explodes', async () => {
```

Comment:
What if we wrap this `it` in a `describe` to describe the context and the use the `beforeEach` to code the context:
```
describe('when the lock update fails', () => {
  beforeEach(() => {
      ;(Collection.update as jest.Mock).mockImplementationOnce(() => {
        throw new Error(errorMessage)
      })
  })
})
```
We can later have the `it` describe only what is expected from the context and the execution of the request:
```
it('it should respond with a 500 and and error message saying that the collection couldn't be updated'
```

We can also do the same for the other case, specifying that the correct response will occur whenever the update works.

Reply:
Beware that you have an async closure here but there's no await.